### PR TITLE
[ci skip] Fix outdated class in multiple database docs.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: guides/source/active_record_multiple_databases.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ that were connected via `connects_to` with the role name of `reading`.
 Note that `connected_to` with a role will look up an existing connection and switch
 using the connection specification name. This means that if you pass an unknown role
 like `connected_to(role: :nonexistent)` you will get an error that says
-`ActiveRecord::ConnectionNotEstablished (No connection pool with 'AnimalsBase' found
+`ActiveRecord::ConnectionNotEstablished (No connection pool with 'ActiveRecord::Base' found
```
Comment (tgxworld) [inline guides/source/active_record_multiple_databases.md:314]: `AnimalsBase` has been removed from the docs for sometime.

Comment (vipulnsward) [inline guides/source/active_record_multiple_databases.md:314]: Should be:

ActiveRecord::ConnectionNotEstablished (No connection pool for 'ActiveRecord::Base' found for the 'non_existent' role.)

https://github.com/rails/rails/blob/31461d8a79cf7a6455589319b2e2d6fac76a04c1/activerecord/test/cases/connection_adapters/connection_handlers_multi_db_test.rb#L378

Comment (tgxworld) [reply guides/source/active_record_multiple_databases.md]: Good catch :+1:  I've updated the PR though I wonder if we should change the doc to be more generic and less coupled to the actual error message.

**Comment by eileencodes** [comment]
Backported in 84742f1d5de0c375ffbe9886a63e4c81c7b7a841

**Pull request merged**